multipart_id_numbering.conf

```diff
@@ -0,0 +1,311 @@
+#
```

Comment:
Maybe just a comment that says "see `multipart_id_numbering.conf`" in the base configuration directory in `providence`?